library/src/interactor_impl.cxx

```diff
@@ -558,16 +498,92 @@ class interactor_impl::internals
     this->Window.render();
   }
 
+  struct tokenize_exception : public exception
+  {
+    explicit tokenize_exception(const std::string& what = "")
+      : exception(what)
+    {
+    }
+  };
+
+  static std::vector<std::string> Tokenize(std::string_view str)
```

Comment:
Maybe `Tokenize` could be moved in `f3d::utils::tokenize`? While it's not reused yet, maybe it will at some point?  
And I think it's a great candidate for fuzz testing to properly test it so we may need to access it from the test framework.

Reply:
I agree to put into utils, but Ill not implement fuzz testing in this PR